composer/core/evaluator.py

```diff
@@ -18,12 +18,14 @@
 __all__ = ["Evaluator", "evaluate_periodically", "ensure_evaluator"]
 
 
-def evaluate_periodically(eval_interval: Union[str, Time, int]):
+def evaluate_periodically(eval_interval: Union[str, Time, int], eval_at_fit_end: bool = True):
     """Helper function to generate an evaluation interval callable.
 
     Args:
         eval_interval (str | Time | int): A :class:`.Time` instance or time string, or integer in epochs,
             representing how often to evaluate. Set to ``0`` to disable evaluation.
+        eval_at_fit_end (bool): Whether to evaluate at the end of training, regardless of `eval_interval`.
```

Comment:
Can you also update the trainer and evaluator docstrings to denote that evaluation always happens at the end of training when specifying a time string? 

https://github.com/mosaicml/composer/blob/e85302b292bf4857e956e2844713afd61558c302/composer/trainer/trainer.py#L355

https://github.com/mosaicml/composer/blob/0f1624ed3d36aa7c032209f115b3af9ec9e0402d/composer/core/evaluator.py#L85

Reply:
Done!